### PR TITLE
Fixed issue with comma after loglevel

### PR DESCRIPTION
**Pull request by brunowego**
:+1:

### CODE REVIEW

**Comment by brunowego** [comment]
CI fail on "jshint:all" task.

File: app/templates/scripts/requirejs/config.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var require = {
         app: '../app/Resources/public/scripts/app',
         modules: '../app/Resources/public/scripts/modules',
         jquery: 'jquery/dist/jquery',
-        loglevel: 'loglevel/dist/loglevel.min'<% if (useFoundation) { %>,
+        loglevel: 'loglevel/dist/loglevel.min',<% if (useFoundation) { %>
```
Comment (bezoerb) [inline app/templates/scripts/requirejs/config.js]: nice catch. 
The comma needs to be inside the conditional block so you won't hit an error when using something other than `foundation` or `bootstrap`

File: app/templates/scripts/requirejs/config.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ var require = {
         'foundation/slider': 'foundation/js/foundation/foundation.slider',
         'foundation/tab': 'foundation/js/foundation/foundation.tab',
         'foundation/tooltip': 'foundation/js/foundation/foundation.tooltip',
-        'foundation/topbar': 'foundation/js/foundation/foundation.topbar'<% } else if (useBootstrap && !useSass) { %>,
+        'foundation/topbar': 'foundation/js/foundation/foundation.topbar',<% } else if (useBootstrap && !useSass) { %>
```
Comment (bezoerb) [inline app/templates/scripts/requirejs/config.js]: results in a comma after the last line

``` javascript
   ...
   'foundation/topbar': 'foundation/js/foundation/foundation.topbar',
}

```

File: app/templates/scripts/requirejs/config.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ var require = {
         'bootstrap/scrollspy': 'bootstrap/js/scrollspy',
         'bootstrap/tab': 'bootstrap/js/tab',
         'bootstrap/tooltip': 'bootstrap/js/tooltip',
-        'bootstrap/transition': 'bootstrap/js/transition'<% } else if (useBootstrap) { %>
+        'bootstrap/transition': 'bootstrap/js/transition',<% } else if (useBootstrap) { %>
```
Comment (bezoerb) [inline app/templates/scripts/requirejs/config.js]: results in a comma after the last line

``` javascript
   ...
   bootstrap/transition': 'bootstrap/js/transition',
}
```

File: app/templates/test/requirejs/test-main.js [hunk 4]
```diff
@@ -20,7 +20,7 @@ require.config({
         app: '../app/Resources/public/scripts/app',
         modules: '../app/Resources/public/scripts/modules',
         jquery: 'jquery/dist/jquery',
-        loglevel: 'loglevel/dist/loglevel.min'<% if (useFoundation) { %>,
+        loglevel: 'loglevel/dist/loglevel.min',<% if (useFoundation) { %>
```
Comment (bezoerb) [inline app/templates/test/requirejs/test-main.js]:
``` javascript
loglevel: 'loglevel/dist/loglevel.min'<% if (useFoundation) { %>,
```

File: app/templates/scripts/requirejs/config.js (hunk 2 above)
Comment (bezoerb) [inline app/templates/scripts/requirejs/config.js]: use

``` javascript
'foundation/topbar': 'foundation/js/foundation/foundation.topbar'<% } else if (useBootstrap && !useSass) { %>,
```

File: app/templates/scripts/requirejs/config.js (hunk 3 above)
Comment (bezoerb) [inline app/templates/scripts/requirejs/config.js]:
``` javascript
'bootstrap/transition': 'bootstrap/js/transition'<% } else if (useBootstrap) { %>,
```

File: app/templates/test/requirejs/test-main.js [hunk 5]
```diff
@@ -38,7 +38,7 @@ require.config({
         'foundation/slider': 'foundation/js/foundation/foundation.slider',
         'foundation/tab': 'foundation/js/foundation/foundation.tab',
         'foundation/tooltip': 'foundation/js/foundation/foundation.tooltip',
-        'foundation/topbar': 'foundation/js/foundation/foundation.topbar'<% } else if (useBootstrap && !useSass) { %>,
+        'foundation/topbar': 'foundation/js/foundation/foundation.topbar',<% } else if (useBootstrap && !useSass) { %>
```
Comment (bezoerb) [inline app/templates/test/requirejs/test-main.js]:
``` javascript
'foundation/topbar': 'foundation/js/foundation/foundation.topbar'<% } else if (useBootstrap && !useSass) { %>,
```

File: app/templates/test/requirejs/test-main.js [hunk 6]
```diff
@@ -51,7 +51,7 @@ require.config({
         'bootstrap/scrollspy': 'bootstrap/js/scrollspy',
         'bootstrap/tab': 'bootstrap/js/tab',
         'bootstrap/tooltip': 'bootstrap/js/tooltip',
-        'bootstrap/transition': 'bootstrap/js/transition'<% } else if (useBootstrap) { %>
+        'bootstrap/transition': 'bootstrap/js/transition',<% } else if (useBootstrap) { %>
```
Comment (bezoerb) [inline app/templates/test/requirejs/test-main.js]:
``` javascript
'bootstrap/transition': 'bootstrap/js/transition'<% } else if (useBootstrap) { %>,
```

**Comment by brunowego** [comment]
Thanks @bezoerb :+1:

**Comment by bezoerb** [comment]
LGTM :+1: 
@brunowego thanks for working on this

**Pull request merged**